include/battle_util.h

```diff
@@ -236,5 +236,6 @@ bool8 ChangeTypeBasedOnTerrain(u8 battlerId);
 void RemoveConfusionStatus(u8 battlerId);
 u8 GetBattlerGender(u8 battlerId);
 bool8 AreBattlersOfOppositeGender(u8 battler1, u8 battler2);
+u32 CalcMoveSecondaryEffectChance(u8 battlerId, u8 secondaryEffectChance);
```

Comment:
In my opinion, either rename it to `CalcSecondaryEffectChance` or pass in `u16 move` as the second argument. Right now the API is confusing because someone unfamiliar with the engine would think they had to pass in a move.

Reply:
The latter is not viable, so I'll do the former.